backend/api/Areas/Admin/Models/Parcel/ParcelModel.cs

```diff
@@ -1,39 +1,14 @@
 using System.Collections.Generic;
-using Model = Pims.Api.Models;
 
 namespace Pims.Api.Areas.Admin.Models.Parcel
 {
-    public class ParcelModel : Model.BaseModel
+    public class ParcelModel : PropertyModel
```

Comment:
Parcel and Building will now inherit from Property so we can manage shared properties.